docs/reference/contrib/modeladmin/index.rst

```diff
@@ -48,6 +48,7 @@ Want to know more about customising ``ModelAdmin``?
     create_edit_delete_views
     inspectview
     chooseparentview
+    reversing_urls
```

Comment:
I think it might be a good idea to create a new section here, rather than having the `reversing_urls` page at the same level. I can easily imagine other helpful pages being added in future, that might fall better into a `tips-and-tricks` or section - open to suggestions on the naming there :). What do you think @lb-?

Reply:
yeah good point, this is not really part of `Want to know more about customising ModelAdmin?` - I have put a new section at the end of the page, see what you think